retirement/models.py

```diff
@@ -142,6 +142,48 @@ class Meta:
         default=False
     )
 
+    google_maps_url = models.CharField(
+        blank=True,
+        null=True,
+        max_length=2000,  # Max URL length supported by IE
+        verbose_name=_("Google maps URL"),
+    )
+
+    accessibility_detail = models.TextField(
+        verbose_name=_("Accessibility Detail"),
+        null=True,
+        blank=True,
+    )
+
+    sub_title = models.CharField(
+        blank=True,
+        null=True,
+        max_length=2000,  # Max URL length supported by IE
+        verbose_name=_("Sub Title"),
+    )
+
+    description = models.TextField(
+        verbose_name=_("Email content"),
```

Comment:
Please change this `verbose_name`, `description` is not for email but for UI of website